src/molecules/formfields/SelectField/Readme.md

```diff
@@ -4,9 +4,9 @@ Label Example:
       label='Select Field Label'
       placeholder='Placeholder'
       selectOptions={[
-        { value: 'Option 1' },
-        { value: 'Option 2' },
-        { value: 'Option 3' }
+        { label: 'Option 1', value: 1 },
```

Comment:
@cisacke however, I think this changes up the `SelectCard` example, if I'm not mistaken, since it's wired to set a `value` state based on the value prop for the option, which then gets displayed to the user.